client/components/FullStory.js

```diff
@@ -1,16 +1,49 @@
 // eslint-disable-next-line no-undef
 import CenteredImage from './sections/CenteredImage';
 
-const {Container, Row, Col} = ReactBootstrap
+const {Container, Row, Col, Button} = ReactBootstrap
 import * as Scroll from 'react-scroll'
 import Markdown from 'react-markdown-it'
 
 
 import Base from './Base'
-import Ab from './Ab'
+import Ab from './Ab' 
 
 export default class FullStory extends Base {
 
+  constructor(props) {
+    super(props);
+
+    this.state = {email: ''};
+
+    this.bindMany([
+      'handleChange',
+      'handleSubmit'
+    ])
+  }
+
+  handleChange(event) {
+    this.setState({email: event.target.value});
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+    this.Store.email = this.state.email; 
+
+    if (!this.state.email) {
+      alert("Please enter an email.");
+      return
+    }
+
+    if (localStorage.getItem(this.state.email)) {
+      alert("You have already subscribed!");
+      return
+    } else {
+      localStorage.setItem(this.state.email, this.state.email);
```

Comment:
Here you access the local storage but this way, if other components needs to know about it, they will never know. If you save the data in the global Store, any other component will re-render and update with the change. In this case it is not necessary, but in a general case it could be and that is the way to go